conf/settings.py

```diff
@@ -68,6 +68,7 @@
     'export_elements',
     'directory_components',
     'prototype',
+    'marketing',
     'euexit',
```

Comment:
can we be more specific - technically everything in our services is marketing (we're a wholly-owned subsidiary of the marketing department, after all)

Reply:
naming is hard

Reply:
as discucced - can you please move these to `core`